client/components/web-preview/toolbar.jsx

```diff
@@ -141,11 +133,7 @@ PreviewToolbar.defaultProps = {
 };
 
 const mapStateToProps = ( state, ownProps ) => {
-	const site = getSelectedSite( state );
-	const showSEO = ownProps.showSEO && (
-		( site && site.plan && hasBusinessPlan( site.plan ) && isEnabled( 'manage/advanced-seo' ) ) ||
-		( isEnabled( 'manage/advanced-seo' ) && isEnabled( 'manage/advanced-seo/preview-nudge' ) )
-	);
+	const showSEO = ownProps.showSEO;
 
 	return { showSEO };
```

Comment:
`ownProps` already pass through `connect()`. is there a reason to leave this all in right here?